### DISCUSSION TITLE
Mattermost for Blog Post Comments

### DISCUSSION THREAD

**Discussion by aedott** [mattermost/community#66] [Moonshot Ideas]
This is very much a "What if?" kind of idea, but... what if instead of having a normal Comments section on a blog post, we had a place where you could leave a comment/start a discussion about the post in a Mattermost channel? For existing community members it would be an interesting way to bring our content back into the conversations happening on the community server, and for new folks it would be a unique way to introduce them to Mattermost. 

I'm imagining something that looks like the text input box in the Mattermost UI, embedded on the blog page with a button that says "Post to Mattermost."

**Comment by chetanyakan** [comment]
All the blog posts can also be tracked in the same channel, where parent posts pointing to the blog post and replies to that post are comments.

This will be even cooler if the collapsed-reply-threads feature is enabled.

**Comment by darkLord19** [comment]
This is such a great idea. I would definitely integrate it in my blog if it were to be implemented!
